torch/_logging/_registrations.py

```diff
@@ -25,3 +27,14 @@
 register_artifact("perf_hints", off_by_default=True)
 
 register_artifact("custom_format_test_artifact", log_format="")
+
+register_log_group(
+    "default",
+    {
+        "graph_breaks": True,
+        "recompiles": True,
+        "dynamic": logging.INFO,
```

Comment:
So what happens if I add a conflicting log level for dynamic in, e.g., my TORCH_LOGS? What takes precedence?

Reply:
Yeah I thought about this above, I think it would be reasonable to take the min of the conflict? that said I think your proposal below is a lot simpler.